prisma/schema.prisma

```diff
@@ -40,7 +40,7 @@ model Certificate {
   id           Int               @id @default(autoincrement())
   username     String
   domain       String
-  orderUrl     String            @unique @db.VarChar(255)
+  orderUrl     String?           @unique @db.VarChar(255)
```

Comment:
I believe it's because in this flow the certificates are created before the orders with only the username and domain and the orders are then added in `order-creator-worker.server.ts`, there's a few steps there like initializing lets encrypt and then actually creating the order that can throw which I imagine might benefit from the job retrying on failure (not sure)? But in any case this is how I see it working and why I think orderUrl is optional, cause we start without it.

Reply:
This is also why in `certificate.server.ts` you can see orderUrl moved from create to update

Reply:
@sfrunza13 you are spot on!
Also since parent queues can only access **direct children's return values** I needed a way to pass all jobs the certificate id, and I can only do that if I create it in the flow, where I don't yet have the order URL